### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by djmtype** [nanostudio-org/nanogallery2#143]
[http://nanogallery.brisbois.fr](http://nanogallery.brisbois.fr) seems [ADA](http://www.interactiveaccessibility.com/services/ada-compliance)-friendly, so that makes me wonder why Nanogallery isn't as keyboard-friendly. For example, It's not possible to navigate among the albums or gallery's breadcrumbs via keyboard.

This might be out of this project's scope so I'd thought I'd ask anyway if Nanogallery intends on supporting keyboard accessibility?

**Comment by Kris-B** [comment]
I don't know how to implement such a feature because this gallery may only be a part of a page/application:
- it may conflict with other keyboard management implementations
- multiple galleries may implemented on the same page

ADA is important but not easy to implement here.

**Comment by djmtype** [comment]
Thanks for getting back to me @Kris-B

**Comment by Kris-B** [comment]
I need a little bit time to find a solution....

**Comment by bebo22** [comment]
I agree with this request, not only in the keyboard management but also in making the elements focusable this library uses div everywhere, which is not ADA-friendly, you should use buttons for action items
Also, pick a key for pause and play I would recommend the **space**
**Enter** should be preserved for actions only.
Also, since it is a lightbox the focus should remain trapped within the lightbox as you can see here https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/

I like the gallery but needs a bit more ADA improvement. keep up the good work! and thanks.
